apiary.apib

```diff
@@ -1017,6 +1017,50 @@ API endpoints that Operation Code's Rails backend makes available to its React f
             {
                 errors: "Some error message"
             }
+## Sessions | Create [/api/v1/sessions/user{?email,password}]
++ Parameters
+
```

Comment:
`user` is also passed, so you'll want to include that as a parameter.  For it's type, you can use `object`.